Forte.ContentfulSchema/Core/ContentTypeDefinition.cs

```diff
@@ -74,8 +78,20 @@ public bool Update(ContentType contentType)
                 }
                 else
                 {
-                    this.UpdateField(match.Existing, match.Updated, ref modified); 
+                    UpdateField(match.Existing, match.Updated, ref modified); 
+                }
+            }
+
+            if (InferredContentType.Fields.Count == contentType.Fields.Count)
+            {
+                for (var i = 0; i < InferredContentType.Fields.Count; i++)
+                {
+                    if (InferredContentType.Fields[i]?.Id != contentType.Fields[i]?.Id) // order has changed
+                    {
+                            modified = true;
```

Comment:
When refactoring the tests, comment this out temporarily to make sure the tests still fail when order change is not detected.